.travis.yml

```diff
@@ -18,3 +18,8 @@ r_github_packages:
 after_success:
   - Rscript -e 'covr::codecov()'
 
+r_packages:
```

Comment:
This should not be required if those packages are listed in the DESCRPITION.

Reply:
I was getting a failure from Travis before about it not finding caret. I thought it was because this wasn't being installed because it's only in `Suggests` rather than `Imports`. That's why I put it in there...